modules/openapi-generator/src/main/resources/elixir/connection.ex.mustache

```diff
@@ -7,7 +7,7 @@ defmodule {{moduleName}}.Connection do
   use Tesla
 
   # Add any middleware here (authentication)
-  plug Tesla.Middleware.BaseUrl, "{{{basePath}}}"
+  plug Tesla.Middleware.BaseUrl, Application.get_env(:{{#underscored}}{{appName}}{{/underscored}}, :base_url)
```

Comment:
> config from dependencies won't be included

Are you sure about that? It would mean that every single `config.exs` from every library is useless, no?
Nonetheless, having this fallback makes perfect sense. :-)